### DISCUSSION TITLE
Question with git log

### DISCUSSION THREAD

**Discussion by jfer1990** [community/community#23321] [New to GitHub]
<p>Im learning about git. I noted a difference when I do the same operation with 2 differents reppos, here it goes :</p>
<p>In the first one when I call for a git log, it appears normally as it supposed to do : in this case it shows “commit hash_number author etc… readme.md”</p>
<p>On the other hand when I move to another reppo and I call for a git log it appears exactly the same but with one difference, before the name of my single file created, in this case readme, it appears the word add, something like this: “commit hash_number author etc…  ‘add’ readme.md”</p>
<p>I made the exact same way to push’em into theis corresponding reppos, the only difference I made is where created the first and second reppo, in the first case, it’s located in butbucket, in the second, in github. What could it be? thnks a lot.</p>

**Comment by lesleyanneb** [comment, marked as answer]
<p>Hi jfer1990,</p>
<p>From what you have described it sounds like what you’re seeing is the commit title. Is it possible that the push didn’t include any information in the commit message besides the filename?</p>
<p>For example you might see: </p>
<p><span>commit SHA</span></p>
<p><span>Author: <a href="mailto:XXX@XXX.com">XXX@XXX.com</a></span></p>
<p>Date: <span>  </span>Thu Jun 7 14:40:05 2018 +1000<br>
<em>Commit Message</em> Add <span>readme.md</span></p>
<p>This would correspond to the short format as outlined in the official Git Documenation here:<br>
<a href="https://git-scm.com/docs/git-log" rel="nofollow noopener">https://git-scm.com/docs/git-log</a></p>
<p>I hope this helps. Please let us know if you need anything else!</p>
